source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -3,278 +3,528 @@
-   ``/home/clr/mix/local``. If the RPMs are not all in the local directory, check
-   to make sure that they are indeed valid RPM files and not corrupt.
+   After the tool exits, you should see your RPMs and a repodata directory in
+   :file:`/home/clr/mix/local-yum`. If the RPMs are not all in this 
+   :file:`local-yum` directory, check to make sure that they are indeed valid
+   RPM files and not corrupt.
+
+#. **Add/remove/edit/list the bundles in your mix**. The bundles in your mix are
+   specified in the Mix Bundle List. This list is stored as a flat file called
+   :file:`mixbundles` in the directory defined by the ``VERSIONS_PATH`` variable
+   in :file:`builder.conf`. This file is generated automatically during
+   initialization, and is read from and written to by mixer when you use it to
+   work with bundles.
+
+   You can view what bundles are already in your mix by running:
+
+   .. code-block:: bash
+
+      # mixer bundle list
+
+   This will show you a list of every bundle in your mix. Bundles are capable of
+   including other bundles, and those bundles can themselves include other
+   bundles. When you list the bundles in your mix this way, mixer will
+   automatically recurse through these includes and show you every single bundle
+   that will end up in your mix.
+
+   If you see a bundle in the list that you weren't expecting, odds are it was
+   included by something you added in. To get a better view at how a bundle
+   ended up in your mix, you can pass the :option:`--tree` flag:
+
+   .. code-block:: bash
+
+      # mixer bundle list --tree
+
+   This will print a tree view of your Mix Bundle List, visually showing what
+   each bundle includes.
+
+   Bundles fall into two categories: **upstream** and **local**. Upstream
+   bundles are those provided by Clear Linux. Local bundles are bundles you've
+   created yourself, or edited versions of upstream bundles.
+
+   Upstream bundle definition files are downloaded and cached for you
+   automatically by mixer, and are stored in the :file:`upstream-bundles`
+   directory created in your working directory. Do *not* modify things in this
+   directory; it is simply a mirror for the tool to use. The tool automatically
+   caches the bundles for your configured version of Clear Linux (in your 
+   :file:`upstreamversion` file), and cleans up old versions once they are no
+   longer needed. You can see what upstream bundles are available by running:
 
-#. **Update/Add bundle definitions**. The mixer uses a local clone of the
-   ``clr-bundles`` repo to define bundles for the mix.
+   .. code-block:: bash
 
-   To define your bundles:
+      # mixer bundle list upstream
 
-      #. Navigate to the ``mix-bundles/`` directory.
-      #. Make any needed modifications to the bundle set.
-      #. Commit the result::
+   Local bundle definition files live in the :file:`local-bundles` directory.
+   The locaiton of this directory is specified by ``LOCAL_BUNDLE_DIR`` in your
```

Comment:
@kevincwells : "locaiton" is mispelled.

Reply:
I've fixed this in a forthcoming push. Just waiting to see if there are other typos to fix :)

Reply:
@kevincwells Please push your latest updates and I'll take one final look and fix minor issues for you before merging. If you push your changes in the next couple of hours, I can merge before EoD.

Reply:
@rcaballeromx Great! I just pushed the change that fixed this typo. It is the only change I am currently aware of. Feel free to fix any other minor things like that as you see them :) Thanks!